src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -248,11 +248,15 @@ public function moduleAction(Request $request)
             }
 
             $collection = ModuleCollection::createFrom([$moduleInstance]);
+            $collection = $modulesProvider->setActionUrls($collection);
```

Comment:
Consider not re-defining variable, instead use another maybe

Easier to track if using a debugger